ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -54,6 +54,7 @@ pub enum Error {
     InvalidProof,
 }
 
+#[derive(Debug, PartialEq)]
```

Comment:
if you derive `PartialEq`, you also likely want `Eq`

see the docs for more info:
https://doc.rust-lang.org/std/cmp/trait.PartialEq.html
https://doc.rust-lang.org/std/cmp/trait.Eq.html